src/cli/Cli.ts

```diff
@@ -62,8 +62,14 @@ export class Cli {
     }
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-function
   private constructor() {
+    const options: Options = {
+      text: 'Running command...',
+      // don't show spinner if running tests
```

Comment:
This comment is a bit misleading. `c8 ignore next` will just ignore uncovered lines.